e2e/client/playwright/utils/index.ts

```diff
@@ -50,3 +50,26 @@ export function sleep(ms: number): Promise<void> {
         setTimeout(resolve, ms);
     });
 }
+
+export async function getCellValueByColumTitle(
+    table: Locator,
+    row: Locator,
+    tableHeading: string,
+): Promise<String> {
+    const headers = table.locator('thead tr th');
+    const count = await headers.count();
+    let columnIndex = -1;
+
+    for (let i = 0; i < count; i++) {
+        const text = await headers.nth(i).innerText();
+
+        if (text === tableHeading) {
+            columnIndex = i;
+            break;
+        }
+    }
+
+    const item = row.locator(`td:nth-child(${columnIndex + 1})`);
```

Comment:
If `tableHeading` can't be found, it would return first column. Better throw an error.